chapter_3/Readme.md

```diff
@@ -0,0 +1,4 @@
+`When you use only integers in ruby with the math operators, then it will return only integers.
+Therefore, if you in put 10/3, it will return 3 even though it is really 3.333...  
+If you input 10.0/3.0, ruby will then return floating point integers, and return a more accurate 3.333... (not sure how my 10th's of 10th's of 10th's it will show me)`
```

Comment:
Just FYI, if either of the arguments is floating point, the whole thing is evaluated with floating point math, i.e. `10.0 / 3.0` is equivalent to `10 / 3.0`